pmm-app/tests/QAN/qanPage.js

```diff
@@ -176,41 +177,39 @@ module.exports = {
   },
 
   async expandAllFilter() {
-    for (let i = 0; i < 4; i++) {
+    for (const i in this.filterGroups) {
       // eslint-disable-next-line max-len
-      let numOfElementsFilterCount = await I.grabNumberOfVisibleElements(
-        this.filterGroupCountSelector(this.filterGroups[i])
+      const numOfElementsFilterCount = await I.grabNumberOfVisibleElements(
+          this.showAllLocator(this.filterGroups[i])
       );
-      if (numOfElementsFilterCount === 1) {
+      if (numOfElementsFilterCount) {
         // eslint-disable-next-line max-len
-        I.click(this.filterGroupCountSelector(this.filterGroups[i]));
+        I.click(this.showAllLocator(this.filterGroups[i]));
```

Comment:
nit: can the `// eslint-disable-next-line max-len` be removed? it looks like redundant

Reply:
removed